### PR TITLE
CC6494: Update interpolated-strings.md #41 [AUTOMATED]

### PR DESCRIPTION
**Pull request by carolinacmoravia**
Hi, @BillWagner, 
This proposed file change comes from: https://github.com/dotnet/docs.ko-kr/pull/41/files. 
Could you help to review this and merge this?
Many thanks in advance.

### CODE REVIEW

**Comment by rpetrusha** [comment]
Thanks, @carolinacmoravia. If you've made this change in one or more localized versions, there's an additional change that needs to be made. In the description of the variables on line 22, `date` also needs to be changed to `hours`.
I'll merge this when the build completes, since I made the additional change in this PR.

**Review by rpetrusha — APPROVED** [review]

File: docs/csharp/language-reference/keywords/interpolated-strings.md
Comment (BillWagner) [inline docs/csharp/language-reference/keywords/interpolated-strings.md]: This change introduced a coding error:

```csharp
{hours}:hh
```

should be:

```csharp
{hours:hh}
```

Otherwise, the variable won't be formatted correclty.

File: docs/csharp/language-reference/keywords/interpolated-strings.md
Comment (BillWagner) [inline docs/csharp/language-reference/keywords/interpolated-strings.md]: Same comment as above. This should be:

```csharp
{1:hh}
```

**Review by BillWagner — CHANGES_REQUESTED** [review]
This does improve the clarity, but the change also introduced two coding errors that should be fixed before we merge this.

**Comment by mairaw** [comment]
I've addressed your feedback @BillWagner. Take a look!

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
Thanks @mairaw This is ready to :shipit: when the build finishes.

**Pull request merged**
